object-manipulation-3/main.js

```diff
@@ -0,0 +1,152 @@
+console.log('Lodash is loaded:', typeof _ !== 'undefined');
+
+var players = [];
+for (var playersIndex = 0; playersIndex < 4; playersIndex++) {
+  players.push({ name: `player${playersIndex + 1}`, hand: [] });
+}
+
+var deck = [];
+
+for (var deckIndex = 0; deckIndex < 52; deckIndex++) {
+  deck.push({ rank: '', suit: '' });
+  if (deckIndex < 13) {
+    deck[deckIndex].suit = 'clubs';
+    if (deckIndex < 9) {
+      deck[deckIndex].rank = deckIndex + 2;
+    } else if (deckIndex === 9) {
+      deck[deckIndex].rank = 'jack';
+    } else if (deckIndex === 10) {
+      deck[deckIndex].rank = 'queen';
+    } else if (deckIndex === 11) {
+      deck[deckIndex].rank = 'king';
+    } else if (deckIndex === 12) {
+      deck[deckIndex].rank = 'ace';
+    }
+  } else if (deckIndex < 26) {
+    deck[deckIndex].suit = 'diamonds';
+    if (deckIndex < 22) {
+      deck[deckIndex].rank = deckIndex - 11;
+    } else if (deckIndex === 22) {
+      deck[deckIndex].rank = 'jack';
+    } else if (deckIndex === 23) {
+      deck[deckIndex].rank = 'queen';
+    } else if (deckIndex === 24) {
+      deck[deckIndex].rank = 'king';
+    } else if (deckIndex === 25) {
+      deck[deckIndex].rank = 'ace';
+    }
+  } else if (deckIndex < 39) {
+    deck[deckIndex].suit = 'spades';
+    if (deckIndex < 35) {
+      deck[deckIndex].rank = deckIndex - 24;
+    } else if (deckIndex === 35) {
+      deck[deckIndex].rank = 'jack';
+    } else if (deckIndex === 36) {
+      deck[deckIndex].rank = 'queen';
+    } else if (deckIndex === 37) {
+      deck[deckIndex].rank = 'king';
+    } else if (deckIndex === 38) {
+      deck[deckIndex].rank = 'ace';
+    }
+  } else {
+    deck[deckIndex].suit = 'hearts';
+    if (deckIndex < 48) {
+      deck[deckIndex].rank = deckIndex - 37;
+    } else if (deckIndex === 48) {
+      deck[deckIndex].rank = 'jack';
+    } else if (deckIndex === 49) {
+      deck[deckIndex].rank = 'queen';
+    } else if (deckIndex === 50) {
+      deck[deckIndex].rank = 'king';
+    } else if (deckIndex === 51) {
+      deck[deckIndex].rank = 'ace';
+    }
+  }
+}
+
+var shuffledDeck = _.shuffle(deck);
+
+var winners = [];
```

Comment:
Rather than doing all this in the global scope, you should make this a function so that creating a deck and set of players is repeatable. You may even want to give it a parameter of a list of the players' names that will be playing so that your players aren't all just numbered.